src/unstuck-bridge-tx.md

```diff
@@ -0,0 +1,286 @@
+# How to Unstuck SORA→ETHEREUM Transactions
+
+::: info
+This guide is written by the community member "Frasier Man" and reviewed by the SORA team. If you want thank Fraiser man, please donate to: `5D4pKyFNRjU8yoYKgiH1UqrjzayjCkttt82HsAdwQ6FTsQyv`
+:::
+
+### Use This Guide If:
+
+- Your SORA bridge transaction is stuck on pending or has failed.
+- The "retry" command on the bridge UI is not working/not available.
```

Comment:
Do you think that we should support this paragraph with the link to Polkaswap or the screenshot of the Retry button?

Reply:
I can add for more clarity, yes.